docs/customer-interactive-api.md

```diff
@@ -47,22 +133,25 @@ curl -H "x-livery-api-key: value" https://www.example.com/services/broadcasts/{b
 
 Response example:
 
-[filename](_customer-interactive-api/_example-VideoExtensionBroadcast.md ':include')
+[filename](_customer-interactive-api/_example-Broadcast.md ':include')
 
-### Get broadcast data (push)
```

Comment:
The push api is not removed from the system right?

Reply:
I was a little confused that it was in the list of API calls. I moved it to "Push restricted broadcast data"